components/buttons/basicButton.tsx

```diff
@@ -11,7 +11,7 @@ const BasicButton = ({ children, className }: BasicButtonProps) => {
           color: white;
           font-size: 1.6rem;
           border: none;
-          border-radius: 64px;
+          border-radius: 16%/50%;
```

Comment:
In this case, px are preferred because the width of the button dynamically determined by the width of the text in the button.
To see the different try to change to width of the button to 250 px and see what append to the border radius.

Width: 'border-radius: 16%/50%;'
![image](https://user-images.githubusercontent.com/47606575/155838378-5c0d377f-c4fc-411c-b40b-5fe8cb11363a.png)

width 'border-radius: 64px;':
![image](https://user-images.githubusercontent.com/47606575/155838400-e904917f-3476-4ce4-96aa-7db2d4aa4b02.png)

Reply:
changed